### PR TITLE
Feature/Change: Stand Up/Kneel down Minigame

### PR DESCRIPTION
**Pull request by Ada18980**
Adds the ability to stand up or kneel down while wearing leg cuffs, leg binders, and strait dresses, among other items. The catch is that there is a new minigame for trying to do this without losing your balance. The minigame is not required to do functionality that was present before; it merely unlocks new features. The minigame is harder the more extensively bound you are--it is nearly impossible when fully encased and hooded, but is easy when minimally restrained.

I also added some new checks and made some changes to the "Help her stand up/kneel" checks, in order to make them more consistent and useful.

### CODE REVIEW

File: BondageClub/Scripts/Character.js [hunk 1]
```diff
@@ -43,6 +43,8 @@ function CharacterReset(CharacterID, CharacterAssetFamily) {
...
 		CanWalk: function () { return ((this.Effect.indexOf("Freeze") < 0) && (this.Effect.indexOf("Tethered") < 0) && ((this.Pose == null) || (this.Pose.indexOf("Kneel") < 0) || (this.Effect.indexOf("KneelFreeze") < 0))) },
 		CanKneel: function () { return ((this.Effect.indexOf("Freeze") < 0) && (this.Effect.indexOf("ForceKneel") < 0) && ((this.Pose == null) || ((!CharacterItemsHavePose(this, "LegsClosed")) && (this.Pose.indexOf("Supension") < 0) && (this.Pose.indexOf("Hogtied") < 0)))) },
+		CanKneelAssisted: function () { return ((CharacterItemsHavePose(this, "LegsClosed")) && (this.Effect.indexOf("ForceKneel") < 0) && (this.Effect.indexOf("Mounted") < 0) && ((this.Pose == null) || (((this.Pose.indexOf("Supension") < 0) && (this.Pose.indexOf("Hogtied") < 0))))) },
+		CanStandAssisted: function () { return ((CharacterItemsHavePose(this, "LegsClosed")) && (this.Effect.indexOf("ForceKneel") < 0) && (this.Effect.indexOf("Mounted") < 0) && ((this.Pose == null) || (((this.Pose.indexOf("Supension") < 0) && (this.Pose.indexOf("Hogtied") < 0))))) },
```
Comment (Nina-1474) [inline BondageClub/Scripts/Character.js:47]: I think it'd be better to not duplicate the function and just reuse CanKneelAssisted, even for standing checks, similar to how CanKneel is used.

Comment (Ada18980) [reply BondageClub/Scripts/Character.js]: Thanks, fixed

**Comment by Ada18980** [comment]
I am going to close this since the current implementation of kneeling now allows players to stand up/kneel down without assistance. However, I won't delete the branch because I think the minigame code can be reused, especially for a future struggle minigame (perhaps for something like finding keys in the dark, if the failure condition were taken out and the player simply had to hit a sweet spot in the middle for a certain amount of time)

**Comment by Ada18980** [comment]
@Ben987 I've reopened this as I was wrong about closing it. 
Anyway, I've merged the latest changes to the current version and it should be good to go.

**Comment by Ben987** [comment]
It could be fun to include this one in the next version.  Can you resolve the conflicts and test it again to make sure everything works?  Thanks

**Comment by Ada18980** [comment]
> It could be fun to include this one in the next version. Can you resolve the conflicts and test it again to make sure everything works? Thanks

I merged it and tested it. Minigame still works, button works as expected

**Comment by Ben987** [comment]
Great!  I will test it soon, thanks for that one :)

**Pull request merged**